### PR TITLE
#193

### LINKED ISSUE #193: Add VPolytope
In #137 we added `HPolytope` and in #168 the computation of vertices list through `Polyhedra.jl`. Now we can add a `VPolytope` type that inherits from `AbstractPolytope`.

### PR DESCRIPTION
**Pull request by mforets**
Closes #193.

### CODE REVIEW

**Comment by schillic** [comment]
Looks good so far. Let us use only one out of "polyhedra/polytopes".

**Comment by mforets** [comment]
ready to go if u approve -- i plan to work further on VPolytope and HPolytope as outlined in #155

File: src/VPolytope.jl [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+- `vertices` -- the list of vertices
+"""
+struct VPolytope{N<:Real} <: AbstractPolytope{N}
+    vertices::Vector{Vector{N}}
+end
+# constructor for a VPolytope with empty vertices list
+VPolytope{N}() where {N<:Real} = VPolytope{N}(Vector{N}(0))
+
+# constructor for a VPolytope with no constraints of type Float64
```
Comment (schillic) [inline src/VPolytope.jl]: "constraints" → "vertices"

File: src/VPolytope.jl [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+"""
+function convex_hull(P1::VPolytope, P2::VPolytope; backend=CDDLib.CDDLibrary())
+    Pch = convexhull(polyhedron(P1, backend), polyhedron(P2, backend))
+    return VPolytope(Pch)
+end
+
+"""
+    cartesian_product(P1::VPolytope, P2::VPolytope; [backend]=CDDLib.CDDLibrary())
+
+Compute the Cartesian product of two polytopes in V-representaion.
```
Comment (schillic) [inline src/VPolytope.jl]: "representation"

File: src/VPolytope.jl [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+- `P1`         -- polytope
+- `P2`         -- another polytope
+- `backend`    -- (optional, default: `CDDLib.CDDLibrary()`) the polyhedral
+                  computations backend, see [Polyhedra's documentation](https://juliapolyhedra.github.io/Polyhedra.jl/latest/installation.html#Getting-Libraries-1)
+                  for further information
+
+### Output
+
+The `VPolytope` obtained by the concrete cartesian product of `P1` and `P2`.
```
Comment (schillic) [inline src/VPolytope.jl]: "Cartesian"

**Comment by mforets** [comment]
thanks for the review 👍

**Comment by schillic** [comment]
There is also a doctest failing: The subtypes of `AbstractPolytope` have changed.

**Comment by mforets** [comment]
ouch, too late. i'll fix it now

**Pull request merged**